tools/pipelines/build-docs.yml

```diff
@@ -88,39 +88,54 @@ variables:
   - name: pnpmStorePath
     value: $(Pipeline.Workspace)/.pnpm-store
 
+# TODO: include release/* branches as trigger. Currently, the website docs builds and deploys on each commit.
```

Comment:
Nit: I don't think we necessarily want to restore release branches as triggers for "this pipeline". If we split this into 2, we would have a pipeline that publishes doc model artifacts, which runs on main and release branches. And then we have the pipeline that actually builds the website, which _only_ runs on main.